src/instagram.erl

```diff
@@ -0,0 +1,314 @@
+-module(instagram).
```

Comment:
Let's use the same naming convention as with the `twitter_client.erl`. So either append `_client` or remove it or use a different suffix. The main requirement is that they need to be consistent.